tests/js/spec/views/performance/landing/index.spec.tsx

```diff
@@ -319,8 +296,12 @@ describe('Performance > Landing > Index', function () {
   describe('with transaction search feature', function () {
     it('renders the search bar', async function () {
       addMetricsDataMock();
+
       const data = initializeData({
         features: ['performance-transaction-name-only-search'],
+        query: {
+          field: 'test',
```

Comment:
If you need query data it gets blocked otherwise, we should clean this up later.
```
    if (!eventView.isValid()) {
      return;
    }
 ```